### DISCUSSION TITLE
How to change results coordinates system in pyansys dpf output ?

### DISCUSSION THREAD

**Discussion by js4561207** [ansys/pydpf-core#1480] [Q&A]
I need map my results from **gload CS system** to local **Cylinder coordinate system.**

**Comment by xavicastel** [comment]
I also need this answer. I have found this example in DPF documentation:
[https://dpf.docs.pyansys.com/version/stable/examples/10-mesh_operations/13-nodes_in_local_coordinate_system.html](url)
But it is not working yet for me.
